core/src/main/kotlin/org/evomaster/core/problem/rest/OpenApiAccess.kt

```diff
@@ -91,14 +105,28 @@ object OpenApiAccess {
         return path.toFile().readText()
     }
 
-    private fun connectToServer(openApiUrl: String, attempts: Int): Response {
+    private fun connectToServer(openApiUrl: String, authentication : HttpWsAuthenticationInfo, attempts: Int): Response {
 
         for (i in 0 until attempts) {
             try {
-                return ClientBuilder.newClient()
-                        .target(openApiUrl)
-                        .request(MediaType.APPLICATION_JSON_TYPE)
-                        .get()
+
+                var currentReq = ClientBuilder.newClient()
+                                              .target(openApiUrl)
+                                              .request(MediaType.APPLICATION_JSON_TYPE)
+
+                for(head in authentication.headers) {
+                    currentReq = currentReq.header(head.name, head.value)
+                }
+
+                //currentReq = currentReq.get()
+
+                return currentReq.get()
+
+                //return ClientBuilder.newClient()
+                //        .target(openApiUrl)
+                //        .request(MediaType.APPLICATION_JSON_TYPE)
+                //        .header(authentication.headers[0].name, authentication.headers[0].value)
+                //        .get()
```

Comment:
remove those comments

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one.